server/game.js

```diff
@@ -103,7 +103,7 @@ export default class Game
 	/// Returns an array of hand-views for each player.
 	///  Your own hand contains the cards, others' hands contains just the number of cards
 	#getPlayersHands()
-	{
+	{	;

```

Comment:
whats this cheeky thing?

Reply:
hmm, pass :), now deleted and re-pushed.